unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneResourcesLoadTracker/Tests/SceneResourcesLoadTrackerShould.cs

```diff
@@ -1,100 +1,117 @@
+using System.Collections;
 using DCL;
 using DCL.Controllers;
+using DCL.ECSComponents;
+using DCL.ECSRuntime;
+using DCL.Models;
 using DCL.WorldRuntime;
 using NSubstitute;
+using NSubstitute.Extensions;
 using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
 
 namespace Tests
 {
     public class SceneResourcesLoadTrackerShould
```

Comment:
this test ideally should be agnostic from the ecs version, so the modification must be according to that... mocking the behavior of both system and testing if we got the expected result.
Maybe wrapping both in a class to mock their behavior and using TestCase for each one of them.
please add a new test for the ecs7 specific loading